.circleci/config.yml

```diff
@@ -15,7 +15,8 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: yarn build
-      - run: yarn demotest
-      - run: yarn demobuild
+      - run: yarn test:demo
+      - run: yarn build:demo
       - run: yarn doc
       - run: yarn codecov
+      - run: yarn build:demo:ivy
```

Comment:
shouldn't we test with ivy too, and thus add a `yarn test:demo:ivy` (and what's necessary to make that a valid command of course)?

Reply:
I don't think we currently can :)

Reply:
OK then.